scipy/stats/_hypotests.py

```diff
@@ -984,8 +1233,8 @@ def boschloo_exact(table, alternative="two-sided", n=32):
     - :math:`H_0 : p_1=p_2` versus :math:`H_1 : p_1 \neq p_2`,
       with `alternative` = "two-sided" (default one)
 
-    Boschloo's exact test uses the p-value of Fisher's exact test as a 
-    statistic, and Boschloo's p-value is the probability under the null 
+    Boschloo's exact test uses the p-value of Fisher's exact test as a
+    statistic, and Boschloo's p-value is the probability under the null
```

Comment:
Not sure if this is intentional, but these style fixes doesn't look germane to this PR

```suggestion
    Boschloo's exact test uses the p-value of Fisher's exact test as a 
    statistic, and Boschloo's p-value is the probability under the null 
```

Reply:
They are not, but IMO they are acceptable as it is a reasonable default for an IDE to remove trailing whitespace when saving.